Feature/library playlists style update

ifechukwu123 opened the pull request:
1. Header
     - Create header for marketing site and another header for dashboard site
     - Add navigation to dashboard site from marketing site
     - Make dashboard Header responsive
2. Dashboard homepage
     - Update styling for dashboard 
     - Make responsive
3. Add to Playlist & Add new playlist modals 
     - Remove "add to queue, add to playlist" modal per figma mockups
     - Make responsive

silk-workouts commented:
@ifechukwu123 
Can you please:

-add size property on to <Image> components/elements
-remove (or hide) logout from marketing nav hamburger dropdown

ifechukwu123 commented:
Updates
- Added "sizes" property to Image elements with "fill" property
- Removed logout from marketing header options
- Video Player Page for Playlists
            - Updated styling on all pages to match figma mockups
            - Added ability to sort videos in the playlist

silk-workouts commented:
Merging to dev with understanding.

Video playlist needs iterative fix to maintain which video is currently playing even if video order change with drag and drop.
